Assets/MRTK/Providers/Experimental/WindowsSceneUnderstanding/WindowsSceneUnderstandingObserver.cs

```diff
@@ -131,18 +139,18 @@ public override void Initialize()
             {
                 observerState = ObserverState.GetScene;
             }
+#endif // SCENE_UNDERSTANDING_PRESENT
         }
 
+#if SCENE_UNDERSTANDING_PRESENT
+
         /// <inheritdoc />
         public override void Enable()
         {
```

Comment:
We may want to ensure `base.Enable()` gets called here, to ensure the service state (https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8906) is accurate. And relatedly, ensure that `base.Enable()` is explicitly _not_ called when `SCENE_UNDERSTANDING_PRESENT` is false.

Reply:
Same with `base.Initialize()`